autoscheduler/frontend/src/redux/actions.ts

```diff
@@ -3,3 +3,28 @@
  * return the action in the form that the Redux reducers need, with a string
  * type.
  */
+import Meeting from '../types/Meeting';
+import {
+  ADD_MEETING, REMOVE_MEETING, REPLACE_MEETINGS, SingleMeetingAction, MultiMeetingAction,
+} from './actionTypes';
+
+export function addMeeting(meeting: Meeting): SingleMeetingAction {
+  return {
+    type: ADD_MEETING,
+    meeting,
+  };
+}
+
+export function removeMeeting(meeting: Meeting): SingleMeetingAction {
+  return {
+    type: REMOVE_MEETING,
+    meeting,
+  };
+}
+
+export function replaceMeetings(meetings: Meeting[]): MultiMeetingAction {
+  return {
+    type: REPLACE_MEETINGS,
+    meetings,
+  };
+}
```

Comment:
While I think the other two are self explanatory, I think there should be a description for this, and mention when this would be used / what situations. I assume it's for quickly changing out meetings when schedules are swapped?

Reply:
Yes, essentially.

Reply:
Yes